ui/index.d.ts

```diff
@@ -198,42 +180,49 @@ export interface IGenericTreeItemOptions {
     iconPath?: string | Uri | { light: string | Uri; dark: string | Uri };
     commandId?: string;
     contextValue: string;
+
+    /**
+     * If true, the tree item picker will execute `commandId`, refresh the tree, and re-prompt at the same level of the tree.
+     * If `commandId` is not defined, it will throw an error.
+     */
+    includeInTreeItemPicker?: boolean;
```

Comment:
I don't understand the comments here.  In what scenario would you execute a command or refresh?

Reply:
So far this is used for "Sign in to Azure...", "Create an Account...", "Select Subscriptions...", and "Attach a Database Account..."

All of those actions will basically "populate" this level of the tree, which is why we execute the command and then refresh that level of the tree.

Reply:
Added a note